Libraries/Renderer/src/renderers/native/ReactNativeBaseComponent.js

```diff
@@ -119,6 +119,10 @@ ReactNativeBaseComponent.Mixin = {
     this.updateChildren(nextElement.props.children, transaction, context);
   },
 
+  getName() {
+    return this.constructor.displayName || 'Unknown';
```

Comment:
Let's do `return this.constructor.displayName || this.constructor.name || 'Unknown';`